File: Assignment_33/Assignment_33_4.java
import java.util.Scanner;

class Alphabate
{
 public boolean checkAlphabate(char ch)
{
    
        if((ch >= 'a')&& (ch <= 'z'))
        {
            return true;

        }
        else
        {
            return false;
        }
        
}
}
class Assignment_33_1
{
    public static void main(String A[])
    {
         char cValue = '\0';
        boolean bRet = false;

        Scanner sobj = new Scanner(System.in);
    
        System.out.println("Enter Character :");
        cValue = sobj.next().charAt(0);

        Alphabate aobj = new Alphabate();

        bRet = aobj.checkAlphabate(cValue);

        if(bRet == true)
        {
            System.out.println("It is Small character ");
        }
        else
        {
            System.out.println("It is not Small Characater");
        }
    
    }
}


   
  
    

File: Assignment_33/Assignment_33_4.c
#include<stdio.h>
#define TRUE 1
#define FALSE 0

typedef int BOOL;
BOOL CheckAlphabate(char ch)
{
    
        if((ch >= 'a')&& (ch <= 'z'))
        {
            return 1;

        }
        else
        {
            return 0;
        }
        
}

int main()
{
    char cValue = '\0';
    BOOL bRet = FALSE;
    
    printf("Enter String :\n");
    scanf("%c",&cValue);
      
    bRet = CheckAlphabate(cValue);

    if(bRet == TRUE)
    {
        printf("It is Small character");
    }
    else
    {
        printf("It is not Small character");
    }
  
    return 0;
}